gammapy/modeling/models/spatial.py

```diff
@@ -439,22 +450,34 @@ def evaluate(lon, lat, lon_0, lat_0, sigma, e, phi):
         exponent = -0.5 * ((1 - np.cos(sep)) / a)
         return u.Quantity(norm * np.exp(exponent).value, "sr-1", copy=False)
 
-    def to_region(self, **kwargs):
-        """Model outline (`~regions.EllipseSkyRegion`)."""
+    def to_region(self, n_sigma=1, **kwargs):
+        """Model outline at a given number of :math:`\sigma`.
+        
+        Parameters
+        ----------
+        n_sigma : float
+            Number of :math:`\sigma` (Default is 1). 
+
+        Returns
+        -------
+        region : `~regions.EllipseSkyRegion`
+            Model outline.
+        """
+
         minor_axis = Angle(self.sigma.quantity * np.sqrt(1 - self.e.quantity ** 2))
         return EllipseSkyRegion(
             center=self.position,
-            height=2 * self.sigma.quantity,
-            width=2 * minor_axis,
+            height=2 * n_sigma * self.sigma.quantity,
+            width=2 * n_sigma * minor_axis,
             angle=self.phi.quantity,
             **kwargs,
         )
 
     @property
     def evaluation_region(self):
-        """Evaluation region"""
+        """Evaluation region consistent with evaluation radius """
         region = self.to_region()
```

Comment:
To really make sure that it's consistent with the `evaluation_radius`, maybe refactor to `return self.to_region(self.evaluation_radius / self.sigma.quantity)`?